block.json

```diff
@@ -17,6 +17,12 @@
 		"freeShippingFrom": {
 			"type": "number",
 			"default": "50"
+		},
+		"messageColor": {
+			"type": "string"
+		},
+		"progressColor": {
+			"type": "string"
```

Comment:
This alone isn't enough. You're not just saving one form of colors (which is just the name here). Your attributes should be two fold: color, and customColor. So, `messageColor`, `customMessageColor`. This is what we did in our blocks.

Reply:
I removed `customMessageColor` and `customProgressColor` as the color settings are working fine on my end without these definitions. I'll add them again, but it would be nice to take a look at my local site to find out why the code works as expected on my end.